examples/scaling-criteo/02-ETL-with-NVTabular.ipynb

```diff
@@ -309,7 +300,7 @@
     "\n",
     "# Deploy a Single-Machine Multi-GPU Cluster\n",
     "protocol = \"tcp\"  # \"tcp\" or \"ucx\"\n",
-    "NUM_GPUS = [0, 1, 2, 3, 4, 5, 6, 7]\n",
+    "NUM_GPUS = [0] #[0, 1, 2, 3, 4, 5, 6, 7]\n",
```

Comment:
Did you just run this on a single gpu? Should we revert this line?

 Looks like the fit times are about 2x previous - and the transform time is about 4.5x previous below